app/plugins/Files/index.html

```diff
@@ -132,6 +138,39 @@
 				</div>
 			</div>
 		</div>
+		<!-- Sliding New Directory Frame -->
+		<div class='frame hidden' id='add-dir'>
+			<div class='header'>
+				<div class='button' id='back-dir'>
+					<i class='fa fa-chevron-left'></i>
+				</div>
+				<div class='title'>
+					Upload Directory
+				</div>
+			</div>
+
+			<div class='pure-g step' id='step1'>
+				<div class='pure-u-1-3 button' id='upload-dir-choice'>
```

Comment:
`pure-u-1-3` was just chosen for the "Add File" frame because there were three options. This looks like it would just be 1 option with 2/3 of the space unoccupied. If you want to keep it as a separate menu, this should be `pure-u-1-1` right?

Otherwise You might also just incorporate it into the other frame and make all the step1 options `pure-u-1-4`